### PR TITLE
Feature/configure dashboard storybook

### PR DESCRIPTION
**Pull request by yury-klika**
Add Dashboard and individual widgets to Storybook

### CODE REVIEW

**Comment by miraage** [comment]
How can we improve imports/paths? Tons of `..` is a smell

**Comment by Drache93** [comment]
> How can we improve imports/paths? Tons of `..` is a smell

What's the module system in Desktop that allows everything to be imported directly? i.e. "my-folder" instead of "../../my-folder"

File: packages/bootstrap/src/components/dashboard/widgets/Demo.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    },
+  },
+  (props) => {
+    if (props.size) {
+      return {
+        height: `${props.size}em`,
+      };
+    }
+
+    return {};
```
Comment (Drache93) [inline packages/bootstrap/src/components/dashboard/widgets/Demo.tsx]: We do this pattern a lot, probably worth a helper to keep things clean.

**Comment by miraage** [comment]
@Drache93 https://create-react-app.dev/docs/importing-a-component/#absolute-imports

**Comment by miraage** [comment]
@Drache93 it uses webpack's `resolve.modules` feature

File: packages/bootstrap/src/components/dashboard/widgets/Demo.tsx (hunk 1 above)
Comment (yury-klika) [reply packages/bootstrap/src/components/dashboard/widgets/Demo.tsx]: > We do this pattern a lot, probably worth a helper to keep things clean.

@Drache93 could you point me to other instances of such approach? I can't find it in `Dashboard`

**Comment by yury-klika** [comment]
Code was merged from `dev` branch and new approach applied to `Storybook` `Dashboard`-related stories

**Comment by i-kushlianski** [comment]
Would be nice to figure out whether we really need the bundled code in our version control

**Pull request merged**